docs/content/any/learn/roles/index.md

```diff
@@ -40,10 +40,10 @@ allow(actor, action, resource) if
 
 {{% callout "Adding roles to your application?" "blue" %}}
 
-Oso Roles provides built in support for role configuration, role data
+Oso Roles provides built-in support for role configuration, role data
 management and more.
```

Comment:
Only `sqlalchemy-oso`'s Roles feature provides built-in support for role data management